versions.txt

```diff
@@ -0,0 +1,5 @@
+behat/behat               v3.11.0            Scenario-oriented BDD framework for PHP
```

Comment:
I wonder why `behat/behat` is listed. Seems it should exclude dev dependencies.

Reply:
It looks like `composer show` has a `--no-dev` flag for when that's the desired behavior.